Terminal.Gui/Views/Toplevel.cs

```diff
@@ -1,3 +1,5 @@
+#define UNIX_KEY_BINDINGS
```

Comment:
I do not want all those other "unix" key bindings enabled on non-Unix platforms. They get in the way of normal usage. They get in the way.

Until https://github.com/gui-cs/Terminal.Gui/issues/3023 is addressed (or something similarly flexible), can you please do this in this PR instead of `#define UNIX_KEY_BINDINGS`

```cs
if (Application.Driver is CursesDriver) 
{
   KeyBindings.Add (Key.Z.WithCtrl, Command.Suspend);
}
```

Thanks.

Reply:
I changed to a conditional `PlatformID.Unix` because now `NetDriver` is also using the `Suspend` method on a `Linux` or `Mac`.

```cs
        if (Environment.OSVersion.Platform == PlatformID.Unix)
        {
            KeyBindings.Add (Key.Z.WithCtrl, Command.Suspend);
        }

```

Reply:
Thanks!